tools/pv_tree_generator/configmodule.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
```

Comment:
I think you don't need to add this module. since only need to call the staging api, can use put the api root in the code, and call secret manager to retrieve the api key

Reply:
done.